pages/markdown.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react'
+import { css } from 'glamor'
+import withData from '../lib/apollo/withData'
+import withT from '../lib/withT'
+
+import Frame from '../components/Frame'
+
+import {
+  colors,
+  CommentBodyBlockQuote,
+  CommentBodyCode,
+  CommentBodyHeading,
+  CommentBodyList,
+  CommentBodyListItem,
+  CommentBodyParagraph,
+  Editorial,
+  Interaction,
+  HR
+} from '@project-r/styleguide'
+
+import { PUBLIC_BASE_URL, CDN_FRONTEND_BASE_URL } from '../lib/constants'
+
+const { P, A, Emphasis, Cursive, StrikeThrough } = Editorial
+
+const styles = {
+  table: css({
+    '& th': {
+      borderBottom: `1px solid ${colors.divider}`,
+      padding: '0 40px 20px 0',
+      textAlign: 'left'
+    },
+    '& td': {
+      padding: '20px 40px 0 0'
+    }
+  })
+}
+
+export default withData(
+  withT(({ url, t }) => {
+    const meta = {
+      pageTitle: t('markdown/pageTitle'),
+      title: t('markdown/title'),
+      description: t('markdown/description'),
+      image: `${CDN_FRONTEND_BASE_URL}/static/team/bern.jpg`,
```

Comment:
Here is a better one:
![md.png](https://user-images.githubusercontent.com/410211/39394181-0ac65418-4acf-11e8-89e5-bd3e7f0c134d.png)